recipes/seismic-rna/meta.yaml

```
@@ -0,0 +1,47 @@
+---
+
+about:
+  home: https://github.com/rouskinlab/seismic-rna
+  license: GPL-3.0
```

```suggestion
  license: GPL-3.0-only
```
Please use SPDX identifier